bin/yb-ctl

```diff
@@ -1401,12 +1412,13 @@ class ClusterControl:
             succeeded = False
             try:
                 with open(initdb_log_path, 'w') as initdb_stdout_file:
-                    subprocess.check_call([
+                    proc = subprocess.Popen([
                         initdb_binary_path,
                         '-D', tmp_pg_data_dir,
                         '-U', 'postgres'
                     ], stdout=initdb_stdout_file, stderr=subprocess.STDOUT)
-                succeeded = True
+                    wait_for_proc_report_progress(proc)
+                    succeeded = proc.returncode == 0
```

Comment:
Indentation for these two lines looks off. Shouldn't it be aligned with "with" on line 1414.

Reply:
actually I think this is probably ok, since I do not initialize proc outside of the `with` block.